pkg/sql/schemachanger/scpb/uml/table.puml

```diff
@@ -101,6 +101,10 @@ object RowLevelTTL
 RowLevelTTL :  TableID
 RowLevelTTL :  RowLevelTTL
 
+object TableZoneConfig
+
+TableZoneConfig :  TableID
+
```

Comment:
nit: these uml diagrams an `elements_generated.go` changes belong in the other commit